### PR TITLE
Updates to #112 refactor

### PR DESCRIPTION
**Pull request by jmeas**
Ready for review/merge!

#### Updates

- Adding jQuery to the package.json. I received an error when I tried to load the tests in the browser 'cuz it couldn't find jquery.
- Update README text w. changes
- Fix a failing test

#### To investigate

The following solutions are failing:

- ~~logical operators~~
  - ~~you should be able to work with logical and (**✘ This works on master**)~~ Fixed!
- ~~permutation~~ (https://github.com/rmurphey/js-assessment-answers/issues/33)
  - ~~you should be able to return the nth number in a fibonacci sequence~~
  - ~~you should be able to return the set of all valid combinations of n pairs of parentheses.~~

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ the `data` directory; you can access it at `/data/<filename>.json`.
 
 ### Available dependencies
 
-The repo includes jQuery, Backbone, Underscore, and RequireJS. If there's other
+The repo includes jQuery, Backbone, and Underscore. If there's other
 stuff you'd find useful, you can put it in the `lib` directory.
```
Comment (ashleygwilliams) [inline README.md]: might as well get rid of the idea of a `lib` directory since one does not exist.

Comment (jmeas) [inline README.md]: :+1: Good catch!

Comment (ashleygwilliams) [inline README.md]: maybe tell people to use `npm` to add things if they need? they would also have to add the dep to `app/runner.html` as well as whatever answers they would use it in using node's `require`.

Comment (jmeas) [inline README.md]: I thought about adding that, but decided against adding this to this particular section. It seems to me that the current tools provided are more than enough to solve the current tests.

I could see instructions on adding new tools belonging in the "I don't like this tech stack" / fork this project section, though.

What do you think? :open_mouth:

**Comment by jmeas** [comment]
There were 3 failing tests. Two are [missing answers](https://github.com/rmurphey/js-assessment-answers/issues/33); the other does work on master. Working to fix it now!

**Pull request merged**